### DISCUSSION TITLE
Corsair h100i aio not working

### DISCUSSION THREAD

**Discussion by imofficial1** [Rem0o/FanControl.Releases#642] [Q&A] [answered]
Great Software. All of my case fans are working great with the exception of my aio. I've spent all day trying to find a solution (with no luck).  Will this only work if the fans are connected directly to the motherboard fan header? My h100i is connected to mb via usb. I'd rather replace icue with this software.  Thanks

**Comment by AtA3301** [comment, marked as answer]
> Great Software. All of my case fans are working great with the exception of my aio. I've spent all day trying to find a solution (with no luck). Will this only work if the fans are connected directly to the motherboard fan header? My h100i is connected to mb via usb. I'd rather replace icue with this software. Thanks

Yes, you need an actual fan header. Support for USB devices would need to be done on the LibreHardware side if at all.

**Reply by imofficial1** [reply]
Thanks for letting me know
